### PR TITLE
.github/workflows/fork.yml: fake hercules status for merge queue

### PR DESCRIPTION
**Pull request by zowoq**
Hopefully this will allow us to merge PRs from forks.

Tested in https://github.com/nix-community/infra/pull/794.

### CODE REVIEW

File: .github/workflows/fork.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: "hercules"
+
+on:
+  pull_request:
```
Comment (zowoq) [inline .github/workflows/fork.yml:4]: This actions isn't sensitive so I think `pull_request` should be fine.

Comment (Mic92) [reply .github/workflows/fork.yml:4]: It's possible to make the approval optional afaik via project settings.

Comment (zowoq) [reply .github/workflows/fork.yml:4]: Probably easier to keep this as pull_request_target like the label action.

**Comment by zowoq** [comment]
It does go through hercules via the merge queue but it isn't shown in the PR. 

ci

**Pull request merged**